http/handler/key/handler.go

```diff
@@ -26,9 +26,9 @@ func NewHandler(notaryService *security.NotaryService, edgeManager *edge.Manager
 	}
 
 	h.Handle("/key",
-		notaryService.DigitalSignatureVerification(httperror.LoggerHandler(h.keyInspect))).Methods(http.MethodGet)
+		httperror.LoggerHandler(h.keyInspect)).Methods(http.MethodGet)
```

Comment:
why was this removed?

Reply:
`/key` endpoint is used by other agents who do not have the signature, so remove the verification for it.